packages/app/src/app/pages/Dashboard/Components/Sandbox/SandboxListItem.tsx

```diff
@@ -65,118 +60,140 @@ export const SandboxListItem = ({
       },
     })}
   >
-    <Grid css={{ width: 'calc(100% - 26px - 8px)' }} columnGap={2}>
-      <Column
-        span={[12, 7, 7]}
-        css={{
-          display: 'block',
-          overflow: 'hidden',
-          paddingBottom: 4,
-          paddingTop: 4,
-        }}
-      >
-        <Stack gap={4} align="center" marginLeft={2}>
-          <Stack
-            as="div"
-            ref={thumbnailRef}
-            justify="center"
-            align="center"
-            css={css({
-              borderRadius: 'small',
-              height: 32,
-              width: 32,
-              backgroundSize: 'cover',
-              backgroundPosition: 'center center',
-              backgroundRepeat: 'no-repeat',
-              border: '1px solid',
-              borderColor: 'grays.500',
-              backgroundColor: 'grays.700',
-              flexShrink: 0,
-              position: 'relative',
-              svg: {
-                filter: 'grayscale(1)',
-                opacity: 0.1,
-              },
-            })}
-            style={{
-              [screenshotUrl
-                ? 'backgroundImage'
-                : null]: `url(${screenshotUrl})`,
-            }}
-          >
-            {alwaysOn && (
-              <Tooltip label="Always-On">
-                <span
-                  css={css({
-                    backgroundColor: 'green',
-                    width: 3,
-                    height: 3,
-                    position: 'absolute',
-                    right: '-6px',
-                    bottom: '-4px',
-                    borderRadius: '50%',
-                  })}
-                />
-              </Tooltip>
-            )}
-            {screenshotUrl ? null : <TemplateIcon width="16" height="16" />}
+    <Element
```

Comment:
the only change here is adding this generic `Element`which can be modified from outside of the `SandboxListItem` to be a `link` or a `clickable` element. see the `interactionProps` that are passed conditionally from packages/app/src/app/pages/Dashboard/Components/Sandbox/index.tsx